brian2/devices/cpp_standalone/device.py

```diff
@@ -906,14 +907,45 @@ def write_static_arrays(self, directory):
             arr.tofile(os.path.join(directory, 'static_arrays', name))
             static_array_specs.append((name, c_data_type(arr.dtype), arr.size, name))
         self.static_array_specs = static_array_specs
-            
+    
+    def get_all_objects(self, net):
```

Comment:
This seems to be quite redundant with the code in `Network`, can you not directly call `net._sort_objects()`?

Reply:
Yes, I can. Sorry for that poor one.